pyaedt/modules/report_templates.py

```diff
@@ -2680,6 +2686,29 @@ def _context(self):
             "SimValueContext:=",
             sim_context,
         ]
+        if len(self.expressions) == 1:
+            sid = 0
+            pid = 0
+            expr = self.expressions[0]
+            category = "Eye"
+            found = False
+            while not found:
+                available_quantities = self._post.available_report_quantities(
+                    self.report_category, self.report_type, self.setup, category, arg
+                )
+                if len(available_quantities) == 1 and available_quantities[0].lower() == expr.lower():
+                    found = True
+                else:
+                    sid += 1
+                    pid += 1
+                    arg[2][arg[2].index("SID") + 2] = str(sid)
+                    arg[2][arg[2].index("PID") + 2] = str(pid)
+                # Limited maximum iterations to 1000 (Too many probes to analyze even in a single design)
+                if sid > 1000:
+                    self._post.logger.error(f"Failed to find right context for expression : {self.expressions}")
```

Comment:
@mahesh-dhanekula Here is the error. In Ironpython 2.7, f"{}" did not exist. For that reason we need to use:

"Failed to find right context for expression : {}".format(self.expressions)

Reply:
Been a fan of these **f** strings and overlooked them, will make a commit after modification